js/error-handling.md

```diff
@@ -25,6 +25,50 @@ BAD
 class ExampleError extends Error {}
 ```
 
+#### Define meaningful error classes
```

Comment:
I think this could be merged with `Pass additional, useful information along with the error message` into something like `Define reusable error classes, provide details as error message`

Reply:
That's right, that was my first thought as well, but couldn't figure out a meaningful description.

Thanks!